### PR TITLE
Add ElectronicArts detection support

### PR DESCRIPTION
**Pull request by 0xA50C1A1**
Please sign (check) the below before submitting the Pull Request:

- [x] I have signed the ntop Contributor License Agreement at https://github.com/ntop/legal/blob/main/individual-contributor-licence-agreement.md
- [x] I have read the contributing guide lines at https://github.com/ntop/nDPI/blob/dev/CONTRIBUTING.md
- [x] I have updated the documentation (in doc/) to reflect the changes made (if applicable)

Describe changes:

Almost all such traffic is related to their EA Origin store, but there can be collisions when you're just browsing ea.com.

### CODE REVIEW

File: tests/cfgs/default/result/custom_rules_ipv6.pcapng.out [hunk 1]
```diff
@@ -21,17 +21,17 @@ Patricia risk IPv6:   0/0 (search/found)
 Patricia protocols:   0/0 (search/found)
 Patricia protocols IPv6: 9/5 (search/found)
 
-Unknown	1	318	1
+Unknown	1	1287	1
 CustomProtocolD	2	600	1
 CustomProtocolE	1	1287	1
-CustomProtocolF	1	1287	1
+CustomProtocolG	1	318	1
 CustomProtocolH	1	318	1
```
Comment (IvanNardi) [inline tests/cfgs/default/result/custom_rules_ipv6.pcapng.out:28]: ???? Why these changes?
For example, the session `UDP [247f:855b:5e16:3caf:3f2c:4134:9592:661b]:36098 -> [21bc:b273:7f68:88d7:77a8:585:3990:927b]:50621` changed from `CustomProtocolF` to `CustomProtocolC`!!!!

Do you have any idea why (the issue, if real, seems totally unrelated to your change)?

Comment (0xA50C1A1) [reply tests/cfgs/default/result/custom_rules_ipv6.pcapng.out:28]: > ???? Why these changes? For example, the session `UDP [247f:855b:5e16:3caf:3f2c:4134:9592:661b]:36098 -> [21bc:b273:7f68:88d7:77a8:585:3990:927b]:50621` changed from `CustomProtocolF` to `CustomProtocolC`!!!!
> 
> Do you have any idea why (the issue, if real, seems totally unrelated to your change)?

I dunno,  maybe it's caused by protocol ID incrementation?

Comment (IvanNardi) [reply tests/cfgs/default/result/custom_rules_ipv6.pcapng.out:28]: Found the issue. I will try to push a fix asap

File: tests/cfgs/default/result/electronicarts.pcapng.out [hunk 2]
```diff
@@ -0,0 +1,34 @@
+DPI Packets (TCP):	16	(8.00 pkts/flow)
```
Comment (IvanNardi) [inline tests/cfgs/default/result/electronicarts.pcapng.out]: Instead of adding a new trace (with trivial TLS traffic), could you add these two flows to the existing `sites.pcap` file, please?

**Review by IvanNardi — CHANGES_REQUESTED** [review]
You can ignore the change triggered by the custom rules; I am going to fix that in a separate PR

**Comment by 0xA50C1A1** [comment]
Well, I've fixed that.

**Review by IvanNardi — APPROVED** [review]
Thanks!
